libtransmission/tr-macros.h

```diff
@@ -159,6 +165,9 @@
 ****
 ***/
 
+/* Only use this macro to suppress false-positive alignment warnings */
+#define DISCARD_ALIGN(ptr, type) ((type)(void*)(ptr))
```

Comment:
Maybe rename to `TR_DISCARD_ALIGN`. `SHA_DIGEST_LENGTH` is the only outcase here really which we should probably fix someday too.

Reply:
changed in c11760d3e6b76fe8e00e37fca398e7eafd442510